x-pack/test/security_solution_api_integration/test_suites/detections_response/rule_creation/create_rules.ts

```diff
@@ -127,7 +135,7 @@ export default ({ getService }: FtrProviderContext) => {
         */
         it('should create a single rule with a rule_id and validate it ran successfully', async () => {
```

Comment:
Noted the following behaviour:
If I remove tag `@ess`, but put .only in this file for any test suites:

`yarn detectionResponse:runner:ess` does not find any tests to run, despite there other @ess test suites in other files.
Looks like, test runner takes `only` before tags, when searches for tests suites to run. Not sure, how major this issue can be, but something to note. Maybe it's relatively easy to fix, or it's expected behaviour, although not the way I would like it to run

Reply:
Thank you for sharing!
It's expected and I believe this behavior is intentional and can be useful for focusing on specific tests during development or debugging

Reply:
> It's expected and I believe this behavior is intentional and can be useful for focusing on specific tests during development or debugging

it does not find any test suite to run for a tag @serverless, despite `.only` is used in @ess only. It can be a bit confusing. Maybe worth to be mentioned in README